openapi-generator/templates/go/README.mustache

```diff
@@ -17,7 +17,7 @@ import (
 	"context"
 	"fmt"
 
-	phrase "github.com/phrase/phrase-go/v2"
```

Comment:
I believe this and the next change can be automatized using https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files

Reply:
that's really neat. trying it